offchain/rollups-events/tests/tls.rs

```diff
@@ -0,0 +1,26 @@
+use std::process::Command;
+use test_fixtures::broker_redis::BrokerRedisFixture;
+use testcontainers::clients::Cli;
+
+/// Currently, the redis client we're using does not provide a high-level API to
+/// load certificates, it only loads the ones in the host machine's certificate
+/// store. To avoid having to install certificates in the machine running the
+/// test, it builds and runs a Docker image containing both the Redis server
+/// and a test binary that connects to it, executing the latter after the
+/// container is up.
+#[test_log::test(tokio::test)]
+async fn test_it_connects_via_tls() {
+    let docker = Cli::default();
+    let fixture = BrokerRedisFixture::setup(&docker);
+    let output = Command::new("docker")
+        .arg("exec")
+        .arg(fixture.node.id())
+        .arg("broker-tls-test")
```

Comment:
I think hardcoded arguments should be moved to constants.

`const EXEC_ARG: &str = "exec";`
`const BROKER_ARG: &str = "broker-tls-test";`